src/fcBindingModel.jl

```diff
@@ -8,19 +8,24 @@ function polyfc_ActV(L0, KxStar, f, Rtot::Array, IgGC::Array, Kav::AbstractMatri
     Matrix of size nct * nset filled with ActV/Rbound (depends on if ActI is provided)
     """
     ansType = promote_type(typeof(L0), typeof(KxStar), typeof(f), eltype(Rtot), eltype(IgGC))
-    res = Matrix{ansType}(undef, size(Rtot, 2), size(IgGC, 2))
+    if ActI == nothing
```

Comment:
@cyrillustan I was curious as to why ActI was removed as an input from calcSynergy() when it is still an input to polyfc_ActV. Taking out ActI from all the other functions forces us to only receive binding synergy instead of activity synergy. Other than that, figureB1 seems to be building fine now; it is just the results that look a little bit different.

Reply:
I actually changed `polyfc_ActV()` too so now when ActI is not provided, it will output the prediction of Rmulti

Reply:
So is ActI ever supposed to be provided when performing calcSynergy()?

Reply:
ActI is included in `fit`. If `fit` is not provided, then it should be single-cell activity and still use `murineActI` or `humanActI`. I don't like this actually, but I'm yet to come up with a better way.

Reply:
@cyrillustan Oh okay. I see now where this occurs in calcSynergy(). Sorry I have two more questions though. Is it possible that some of the differences in our plots is dues to line 32-36 of synergy because D1 and D2 are calculated without fit or ActI and then later use fit or ActI to adjust the previously calculated D1 and D2? Also, polyfc_ActV() does not take in fit; all of the functions have been edited to not provide ActI either. So then, why is ActI still an argument to this function? If these are not very straightforward answers, I would be happy to Zoom sometime on the weekend or after 8pm if you have time. Either way works.